src/Sarif.Driver/Sdk/SkimmerBase.cs

```diff
@@ -23,7 +23,9 @@ public SkimmerBase()
 
         abstract protected ResourceManager ResourceManager { get; }
 
-        abstract protected IEnumerable<string> TemplateIds { get; }
+        abstract protected IEnumerable<string> TemplateResourceIds { get; }
+
+        virtual protected IEnumerable<string> RichTemplateResourceIds => new List<string>();
```

Comment:
>RichTemplateResourceIds [](start = 46, length = 23)

See other comment. I like Markdown better. So MarkdownTemplateResourceNames. 

What do you think?

Reply:
The spec uses the term “rich” everywhere to describe messages with formatting information. It starts right in the terminology section, where we define

**rich text message**
    message which contains formatting information such as Markdown formatting characters

… and continues throughout the spec, in the spec language and in all property names such as `result.richMessage`. I think that it’s important for the SDK to use the same language as the spec. Thoughts? (Warning: I feel strongly about this, so I’ll need to be convinced to change it 😊)